tests/release/const.go

```diff
@@ -17,13 +17,15 @@ const (
 	ReleasePipelineServiceAccountDefault string = "release-service-account"
 
 	SourceReleasePlanName          string = "source-releaseplan"
+	SecondReleasePlanName          string = "the-second-releaseplan"
 	TargetReleasePlanAdmissionName string = "demo"
 	ReleasePvcName                 string = "release-pvc"
 	ReleaseEnvironment             string = "production"
 
 	ReleaseCreationTimeout              = 5 * time.Minute
 	ReleasePipelineRunCreationTimeout   = 10 * time.Minute
 	ReleasePipelineRunCompletionTimeout = 20 * time.Minute
+	ReleasePlanStatusUpdateTimeout      = 10 * time.Minute
```

Comment:
this seems awfully high. maybe like 1 minute? if it takes 10 minutes to update a RP status, I'd say we have an issue

Reply:
Updated, thanks!